evergreen.yml

```
@@ -32,6 +32,8 @@ functions:
           if $UPDATE_PLUGIN_VERSION; then
             sed -i "s/\[Version = \"0.0.0\"\]/\[Version = \"${RELEASE_VERSION}\"\]/g" \
               connector/MongoDBAtlasODBC.pq
+            sed -i "s/<connector-version>/\"${RELEASE_VERSION}\"/g" \
```

```suggestion
            sed -i "s/<connector-version>/${RELEASE_VERSION}/g" \
```
Test it out like this.  The way you have it looks like it will be: `"powerbi-connector+"1.0.0"",`